content/docs/create-a-new-react-app.md

```diff
@@ -30,43 +30,29 @@ This is also **the easiest way to integrate React into an existing website.** Yo
 
 The React team primarily recommends these solutions:
 
-- If you're **learning React** or **creating a new [single-page](/docs/glossary.html#single-page-application) app,** use [Create React App](#create-react-app).
-- If you're building a **server-rendered website with Node.js,** try [Next.js](#nextjs).
-- If you're building a **static content-oriented website,** try [Gatsby](#gatsby).
+- If you're building a **static or server-rendered application,** use [Next.js](#nextjs).
+- If you're building a **static content-oriented website,** use [Gatsby](#gatsby).
+- If you're building a **[single-page app](/docs/glossary.html#single-page-application)**, use [Create React App](#create-react-app).
 - If you're building a **component library** or **integrating with an existing codebase**, try [More Flexible Toolchains](#more-flexible-toolchains).
 
-### Create React App {#create-react-app}
-
-[Create React App](https://github.com/facebookincubator/create-react-app) is a comfortable environment for **learning React**, and is the best way to start building **a new [single-page](/docs/glossary.html#single-page-application) application** in React.
-
-It sets up your development environment so that you can use the latest JavaScript features, provides a nice developer experience, and optimizes your app for production. You’ll need to have [Node >= 10.16 and npm >= 5.6](https://nodejs.org/en/) on your machine. To create a project, run:
-
-```bash
```

Comment:
I removed this in favor of linking out to the CRA site, similar to the other frameworks.